commit ff313ee927a160bacbc57468507fd326f5ebf230 (refs/changes/30/4730/2)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2018-08-24T22:24:22+00:00 (1 year, 1 month ago)
    
    MORAY-490 Electric Moray should pass through options for bucket operations

diff --git a/lib/server.js b/lib/server.js
index c019a11..fdaa7fe 100644
--- a/lib/server.js
+++ b/lib/server.js
@@ -481,9 +481,8 @@ function getBucket(options) {
             }
 
             var pnode = hNode.pnode;
-            options.clients.map[pnode].getBucket(bucket,
-                                                 function (gErr, rbucket)
-            {
+            options.clients.map[pnode].getBucket(bucket, opts,
+                function (gErr, rbucket) {
                 log.debug({
                     err: gErr,
                     bucket: rbucket
@@ -573,7 +572,7 @@ function updateBucket(options) {
         }
 
         options.clients.array.forEach(function (client) {
-            client.updateBucket(name, cfg, function (err2) {
+            client.updateBucket(name, cfg, opts, function (err2) {
                 log.debug({
                     err: err2
                 }, 'updateBucket: returned');
@@ -647,7 +646,7 @@ function delBucket(options) {
         }
 
         options.clients.array.forEach(function (client) {
-            client.delBucket(name, function (err2) {
+            client.delBucket(name, opts, function (err2) {
                 log.debug({
                     err: err2
                 }, 'delBucket: returned');
